src/pymatgen/io/lobster/outputs.py

```diff
@@ -1661,16 +1799,23 @@ def get_structure_with_total_grosspop(self, structure_filename: PathLike) -> Str
             Structure Object with Mulliken and Loewdin total grosspopulations as site properties.
         """
         struct = Structure.from_file(structure_filename)
-        mulliken_gps: list[dict] = []
-        loewdin_gps: list[dict] = []
-        for grosspop in self.list_dict_grosspop:
-            mulliken_gps.append(grosspop["Mulliken GP"]["total"])
-            loewdin_gps.append(grosspop["Loewdin GP"]["total"])
-
-        site_properties = {
-            "Total Mulliken GP": mulliken_gps,
-            "Total Loewdin GP": loewdin_gps,
-        }
+        site_properties = {}
```

Comment:
Again: not sure adding thr LCFO prop to a site is even wanted

Reply:
Raises value error now